js/build.js

```diff
@@ -0,0 +1,55 @@
+Fliplet.Widget.instance({
+  name: 'slide',
+  childOf: ['slider'],
+  data: {
+    slideIndex: 0,
+    formCount: 0,
+    sliderRequiredForms: []
+  },
+  render: {
+    class: 'swiper-slide',
+    template: '<div data-view="content"></div>',
+    ready: async function() {
+      var thisSlide = this;
+
+      await Fliplet.Widget.initializeChildren(thisSlide.$el, thisSlide);
```

Comment:
You may want to consider only initialising the children when the slider is slid into view